### PR TITLE
async service update

### PR DESCRIPTION
**Pull request by zyndiecate**
epic https://github.com/giantswarm/giantswarm/issues/242

deps
- https://github.com/giantswarm/user-config/pull/78
- https://github.com/giantswarm/client-go/pull/87
- https://github.com/giantswarm/api/pull/193
- https://github.com/giantswarm/app-client-go/pull/61
- https://github.com/giantswarm/app-service/pull/598

### CODE REVIEW

File: update.go [hunk 1]
```diff
@@ -23,8 +23,13 @@ const (
 	swarm status %s
 
 `
-	updateConfirmUpdateMsg = "Are you sure you want to proceed with the update as described above? [y|n] "
+	updateConfirmUpdateWithPlanMsg = "Are you sure you want to proceed with the update as described above? [y|n] "
+	updateConfirmUpdateQuietMsg    = "Are you sure you want to proceed with the update as described above? [y|n] "
```
Comment (ewoutp) [inline update.go]: Why 2 constants if they are exactly the same?

File: update.go [hunk 2]
```diff
@@ -178,27 +185,40 @@ func updateService(cmd *cobra.Command, definitionFilePath string) error {
 		}
 	}
 
+	if updateDetach {
+		return nil
```
Comment (ewoutp) [inline update.go]: Shouldn't there be some feedback to the user in this case?

**Comment by ewoutp** [comment]
Check with @marians about the user feedback.
Code wise LGTM

File: update.go (hunk 2 above)
Comment (marians) [inline update.go]: What case is it? What's presented so far to the user in this case?

Comment (ewoutp) [inline update.go]: `swarm update -d ...`

@zyndiecate correct me if I'm wrong.

Comment (zyndiecate) [inline update.go]: Correct. We showed nothing. What to do here?

**Comment by zyndiecate** [comment]
I am just merging to get monkeys playing. If there is some ux related stuff, lets fix it!

**Pull request merged**